source/appModules/notepadPlusPlus.py

```diff
@@ -46,7 +46,10 @@ class NppEdit(ScintillaBase.Scintilla):
 	def _get_TextInfo(self):
 		if self.appModule.is64BitProcess:
 			appVerMajor, appVerMinor, *__ = self.appModule.productVersion.split(".")
-			if int(appVerMajor) >= 8 and int(appVerMinor) >= 3:
+			# The minor version of Notepad++ can consist of one or more digits,
+			# e.g. '3' in '8.3', '21' in '8.2.1' or '192' in '8.1.9.2'.

```

Comment:
Hi @leonardder, I'm just going to change this comment and proceed with the merge, hope that's okay.
I was hoping to make the mapping a bit clearer.

```suggestion
			# When retrieving the version, Notepad++ concatenates
			# minor, patch, build in major.minor.patch.build to the form of major.minor
			# https://github.com/notepad-plus-plus/npp-usermanual/blob/master/content/docs/plugin-communication.md#nppm_getnppversion
			# e.g. '8.3' for '8.3', '8.21' for '8.2.1' and '8.192' for '8.1.9.2'.
```